crescent/context/context.py

```diff
@@ -143,14 +145,42 @@ async def respond(
         )
 
         if not (self._has_deferred_response or self._has_created_message):
-            await self.app.rest.create_interaction_response(
-                **kwargs,
-                interaction=self.id,
-                token=self.token,
-                response_type=ResponseType.MESSAGE_CREATE,
-                flags=flags,
-                tts=tts,
-            )
+            if fut := self._unset_future:
+                resp = (
+                    self.interaction.build_response()
+                    .set_content(content)
```

Comment:
I think this section can be a function. I don't really like the indentation.

Reply:
it's only called once, so I'm not sure where the function would really go